.github/workflows/rust.yml

```diff
@@ -45,6 +45,10 @@ jobs:
     - name: Cargo check
       run: cargo check --features "${{ matrix.feature }}" --no-default-features
 
+    - name: Cargo clippy
+      if: ${{ matrix.feature != '' }}
+      run: cargo clippy --no-deps --all-features --all-targets -- -D warnings
```

Comment:
Shouldn't this use the same features as the check build? Or else run outside of the features matrix?

Reply:
I assume that we want to clippy all the code. And running it here is faster, until integration of caching like https://github.com/image-rs/image/pull/2528. I'd do that in a separate PR since it will very probably churn the CI a bit.